config/config.toml

```diff
@@ -0,0 +1,5 @@
+title = "mk8_integrity config"
```

Comment:
Do we need to serialize the title? And, do we in the first place? 

This might cause deserialization to fail. :sweat_smile:

Reply:
Removed